modules/mobile/app/controllers/mobile/v0/letters_controller.rb

```diff
@@ -36,10 +53,15 @@ def beneficiary
         render json: Mobile::V0::LettersBeneficiarySerializer.new(@current_user, response)
       end
 
```

Comment:
I initially created a new route for this because it hits a different endpoint and returns a different data type. But that didn't feel right from the front end perspective. So I talked to front end to confirm that they would prefer a single endpoint with a query param and merged the functionality.